lib/diagnostic.rb

```diff
@@ -45,7 +57,11 @@
 # which returns the string "think think think".
 
 ##
-# your response here
+class Developer < Person
+def initialize(solve_problems)
```

Comment:
This creates a constructor that accepts one argument and sets an instance variable, `@solve_problems`, to it.  What we want is an instance method named `solved_problems`.

```ruby
class Developer < Person
  def solve_problems
    'think think think'
  end
end
```